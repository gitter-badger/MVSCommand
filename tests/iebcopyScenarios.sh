tso alloc dsn\("TEST.COPYSOME"\) recfm\(v,b\) lrecl\(137\) dsorg\(po\) dsntype\(library\) catalog tracks space\(10,10\)
tso alloc dsn\("TEST.COPYALL"\) recfm\(v,b\) lrecl\(137\) dsorg\(po\) dsntype\(library\) catalog tracks space\(10,10\)

mvscmd --pgm=iebcopy --sysprint=* --sysin=ibmuser.test.iebcpy\(copymem\) --sysut2=ibmuser.test.copysome --sysut1=ibmuser.test.c | awk '!/IEB1035I/'
mvscmd --pgm=iebcopy --sysprint=* --sysin=dummy --sysut1=ibmuser.test.c --sysut2=ibmuser.test.copyall | awk '!/IEB1035I/'
tsocmd delete "TEST.COPYSOME"
tsocmd delete "TEST.COPYALL"
