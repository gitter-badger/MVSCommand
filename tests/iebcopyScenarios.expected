alloc dsn(TEST.COPYSOME) recfm(v,b) lrecl(137) dsorg(po) dsntype(library) catalog tracks space(10,10)
alloc dsn(TEST.COPYALL) recfm(v,b) lrecl(137) dsorg(po) dsntype(library) catalog tracks space(10,10)
1                                               IEBCOPY MESSAGES AND CONTROL STATEMENTS                                    PAGE     1
-IEB1135I IEBCOPY  FMID HDZ2210  SERVICE LEVEL UA80390  DATED 20160120 DFSMS 02.02.00 z/OS    02.02.00 HBB77A0  CPU 1090
-  COPY OUTDD=SYSUT2,INDD=SYSUT1
   SELECT MEMBER=(ERR,MAIN)
0IEB1013I COPYING FROM PDSE  INDD=SYSUT1   VOL=B2SYS1 DSN=IBMUSER.TEST.C
 IEB1014I           TO PDSE OUTDD=SYSUT2   VOL=B2SYS1 DSN=IBMUSER.TEST.COPYSOME
 IGW01551I MEMBER ERR  HAS BEEN COPIED
 IGW01551I MEMBER MAIN  HAS BEEN COPIED
 IGW01550I 2 OF 2 SPECIFIED  MEMBERS WERE COPIED
 IEB147I END OF JOB - 0 WAS HIGHEST SEVERITY CODE
1                                               IEBCOPY MESSAGES AND CONTROL STATEMENTS                                    PAGE     1
-IEB1135I IEBCOPY  FMID HDZ2210  SERVICE LEVEL UA80390  DATED 20160120 DFSMS 02.02.00 z/OS    02.02.00 HBB77A0  CPU 1090
-STEP1    COPY      INDD=SYSUT1,OUTDD=SYSUT2         GENERATED STATEMENT
0IEB1013I COPYING FROM PDSE  INDD=SYSUT1   VOL=B2SYS1 DSN=IBMUSER.TEST.C
 IEB1014I           TO PDSE OUTDD=SYSUT2   VOL=B2SYS1 DSN=IBMUSER.TEST.COPYALL
 IGW01551I MEMBER ERR  HAS BEEN COPIED
 IGW01551I MEMBER MAIN  HAS BEEN COPIED
 IGW01550I 2 OF 2  MEMBERS WERE COPIED
 IEB147I END OF JOB - 0 WAS HIGHEST SEVERITY CODE
delete TEST.COPYSOME
IDC0550I ENTRY (A) IBMUSER.TEST.COPYSOME DELETED
delete TEST.COPYALL
IDC0550I ENTRY (A) IBMUSER.TEST.COPYALL DELETED
